rules/php70/tests/Rector/FuncCall/NonVariableToVariableOnFunctionCallRector/Fixture/keep_curly_braces.php.inc

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Rector\Php70\Tests\Rector\FuncCall\NonVariableToVariableOnFunctionCallRector\Fixture;
+
+final class ModelSmokeTest
+{
+    /** @var array|null */
+    private $models;
+
+    public function testGetters(): void
```

Comment:
This looked as test at first sight. To me a while to found out it's a fixture :D

Better use dummy name `run()` as in other tests